package.json

```diff
@@ -1,20 +1 @@
-{
```

Comment:
are these changes required @harshach @chirag-madlani

Reply:
@mohityadav766 those are generated by build

Reply:
I guess it's due to running it form npm instead of yarn, I can update that in this PR to use yarn across the ap